woltka/tools.py

```diff
@@ -223,9 +224,13 @@ def collapse_wf(input_fp:  str,
     if not mapping:
         exit(f'No source-target relationship is found in {basename(map_fp)}.')
 
+    # convert field index
+    if field:
```

Comment:
What is this for?

Reply:
This is change a user-entered field number (starting from 1) into a Python list index (starting from 0).